src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,24 +1,95 @@
 package core.basesyntax;
 
+import java.util.Objects;
 /**
  * <p>Реалізувати свою HashMap, а саме методи `put(K key, V value)`, `getValue()` та `getSize()`.
  * Дотриматися основних вимог щодо реалізації мапи (initial capacity, load factor, resize...)
  * За бажанням можна реалізувати інші методи інтрефейсу Map.</p>
  */
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private int size;
+    private final double loadFactor = 0.75;
+    private final int defaultDefinition = 16;
```

Comment:
Final variables should be named in UPPER_CASE

Reply:
final variables shouldn't be uppercase, constants should